classes/Tools.php

```diff
@@ -4309,6 +4311,46 @@ public static function isFileFresh(string $file, int $timeout = self::CACHE_LIFE
 
         return false;
     }
+
+    /**
+     * @return bool
+     */
+    public static function isCountryFromBrowserAvailable(): bool
+    {
+        $languageAvailable = self::getCountryIsoCodeFromHeader();
+
+        if ($languageAvailable === null) {
+            return false;
+        }
+
+        return Configuration::get('PS_DETECT_COUNTRY') && isset($_SERVER['HTTP_ACCEPT_LANGUAGE'])
```

Comment:
```suggestion
        return Configuration::get('PS_DETECT_COUNTRY')
```

Reply:
This one is redundant since it is checked by `isCountryIsoCodeAvailableInHeader` already

Reply:
done